src/Plugins/Pagination.js

```diff
@@ -162,7 +184,11 @@ class Pagination {
 			result = result.reverse();
 		}
 
-		if (this.data.pagination.filter) {
+		if (
+			this.data.pagination.filter ||
+			this.data.pagination.include ||
+			this.data.pagination.exclude
+		) {
 			result = result.filter((value) => !this.isFiltered(value));
```

Comment:
Is it worth renaming `isFiltered` to `isFilteredOut` or `isExcluded`? Or even inverting the return values and renaming it to `isIncluded`, since then you would have:
```diff
-			result = result.filter((value) => !this.isFiltered(value));
+			result = result.filter((value) => this.isIncluded(value));
```

Reply:
Since main switched to v.3, I think it's okay to rename this function.